app/frontend/dabates/debates.reducers.js

```diff
@@ -0,0 +1,181 @@
+import { 
```

Comment:
I believe this could be encapsulated into a `comments.reducers.js` that's common between `proposals` and `debates`.